codegen/templates/service.tmpl

```diff
@@ -14,24 +14,27 @@ import (
 	module "{{$instance.PackageInfo.ModulePackagePath}}"
 )
 
+// DependenciesTree re-exported for convenience.
+type DependenciesTree module.DependenciesTree
+
 // CreateGateway creates a new instances of the {{$instance.InstanceName}}
 // service with the specified config
 func CreateGateway(
 	config *zanzibar.StaticConfig,
 	opts *zanzibar.Options,
-) (*zanzibar.Gateway, error) {
+) (*zanzibar.Gateway, interface{}, error) {
```

Comment:
for something like interface{}, it at least documents what that interface{} contextually means.